examples/ai-ml/ray/main.tf

```diff
@@ -0,0 +1,365 @@
+provider "aws" {
+  region = local.region
+}
+
+provider "kubernetes" {
+  host                   = module.eks_blueprints.eks_cluster_endpoint
+  cluster_ca_certificate = base64decode(module.eks_blueprints.eks_cluster_certificate_authority_data)
+
+  exec {
```

Comment:
should probably copy the data/token version thats now the standard across examples for consistency - for both Kubernetes and Helm providers (just copy these from another example and paste here)